Dockerfile

```diff
@@ -1,3 +1,37 @@
-FROM starefossen/github-pages:onbuild
+# Get Jekyll build env
+FROM docs/docker.github.io:docs-builder AS builder
 
-CMD jekyll serve -d /_site --watch -H 0.0.0.0 -P 4000
+# Set to the version for this archive
+# You need to set this one more time below too
+ENV VER=v1.4
+
+# Build the docs from this branch
+COPY . /source
+RUN jekyll build --source /source --destination /site/${VER}
+
+# Do post-processing on archive
+COPY --from=docs/docker.github.io:docs-config /scripts/* /usr/bin/
+RUN /usr/bin/fix-archives.sh /site/ ${VER}
+
+# Make an index.html which will redirect / to /${VER}/
+RUN VER=${VER} echo "<html><head><title>Redirect for $VER</title><meta http-equiv=\"refresh\" content=\"0;url='/$VER/'\" /></head><body><p>If you are not redirected automatically, click <a href=\"/$VER/\">here</a>.</p></body></html>" > /site/index.html
```

Comment:
- the `VER=${VER}` is not needed, as the `ENV` (or `ARG`) already is set when running this.
- nit: for consistency, perhaps change all `$VER` to `${VER}`